editor/plugins/animation_blend_tree_editor_plugin.cpp

```diff
@@ -1114,10 +1131,20 @@ AnimationNodeBlendTreeEditor::AnimationNodeBlendTreeEditor() {
 	add_options.push_back(AddOption("TimeSeek", "AnimationNodeTimeSeek", 1));
 	add_options.push_back(AddOption("TimeScale", "AnimationNodeTimeScale", 1));
 	add_options.push_back(AddOption("Transition", "AnimationNodeTransition"));
-	add_options.push_back(AddOption("BlendTree", "AnimationNodeBlendTree"));
-	add_options.push_back(AddOption("BlendSpace1D", "AnimationNodeBlendSpace1D"));
-	add_options.push_back(AddOption("BlendSpace2D", "AnimationNodeBlendSpace2D"));
-	add_options.push_back(AddOption("StateMachine", "AnimationNodeStateMachine"));
+	
+	// Add all types that inherit from AnimationRootNode to options.
+	List<StringName> classes;
+	classes.sort_custom<StringName::AlphCompare>();
+	ClassDB::get_inheriters_from_class("AnimationRootNode", &classes);
```

Comment:
```suggestion
	ClassDB::get_inheriters_from_class("AnimationRootNode", &classes);
	classes.sort_custom<StringName::AlphCompare>();
```
Sorting *before* getting the classes doesn't do anything